### DISCUSSION TITLE
Insert string to different file at line

### DISCUSSION THREAD

**Discussion by UWUplus** [SilentVoid13/Templater#907] [Help] [answered]
Hi, 
I'm looking for a way to insert a string into a different file with specified name when the template is run.
While looking through the discussion I found #612, but it just appends to the file.

Is there a way to do add it at a specific line?

**Comment by AB1908** [comment, marked as answer]
You'd have to replace the file contents with `app.vault.modify()`. Check the Obsidian API docs by Marcus.
